### PR TITLE
Change default Linux CI compiler to gcc-11

### PR DESCRIPTION
**Pull request by KVS85**
oskar PR: https://github.com/arangodb/oskar/pull/400

### CODE REVIEW

**Review by MBkkt — APPROVED** [review]

**Review by fceller — APPROVED** [review]

File: CHANGELOG [hunk 1]
```diff
@@ -1,6 +1,11 @@
 devel
 -----
 
+* Changed default Linux CI compiler to gcc-11.
+
+* Added a fully functional UI for Views that lets users view, modify mutable
+  properties and delete views from the web UI.
+
```
Comment (jsteemann) [inline CHANGELOG:8]: The comment seems unrelated to the changes in this PR

**Review by jsteemann — APPROVED** [review]
LGTM

File: CHANGELOG (hunk 1 above)
Comment (KVS85) [reply CHANGELOG:8]: https://github.com/arangodb/arangodb/commit/b1a216ba4bc3d897815539a8b03cdc9a16fbae3d accidentally deleted 2 CHANGELOG messages. This PR will restore them.
